src/dlgConnectionProfiles.cpp

```diff
@@ -1519,11 +1519,19 @@ void dlgConnectionProfiles::loadProfile(bool alsoConnect)
     if (entries.isEmpty()) {
         firstTimeLoad = true;
     } else {
-        QFile file(qsl("%1%2").arg(folder, profile_history->itemData(profile_history->currentIndex()).toString()));
+        QString fileName{qsl("%1%2").arg(folder, profile_history->itemData(profile_history->currentIndex()).toString())};
+        QFile file(fileName);
         file.open(QFile::ReadOnly | QFile::Text);
         XMLimport importer(pHost);
+
         qDebug() << "[LOADING PROFILE]:" << file.fileName();
-        importer.importPackage(&file, nullptr); // TODO: Missing false return value handler
+        if (auto [success, message] = importer.importPackage(&file, nullptr); !success) {
+            pHost->postMessage(tr("[ ERROR ] - Something went wrong loading your Mudlet profile and it could not be loaded.\n"
+                "Try loading an older version in 'Connect - Options - Profile history' or double-check that %1 looks correct.").arg(fileName));
```

Comment:
Please mention that the file name is expected in %1

Reply:
Oh thanks for adding the comment for translators!
I also meant giving the context to users maybe? 
So they can better verify if filename looks correct.

Reply:
I've played around with quite a few variations and this was the best version that I came up with. Have you tried it out in practice?

Reply:
No I have not, yet.
I was thinking about the random player asked to check if "abcde" is correct, but may not know what is expected.

Reply:
Let's see if that actually happens and folks are getting confused, if they are we can change it. I'd like to keep it simple, short, and concise.